Sources/ComposableArchitecture/Reducer.swift

```diff
@@ -6,11 +6,11 @@ import Combine
 ///
 /// Reducers have 3 generics:
 ///
-/// * `State`: A type that holds the current state of the application
-/// * `Action`: A type that holds all possible actions that cause the state of the application to
-///   change.
-/// * `Environment`: A type that holds all dependencies needed in order to produce `Effect`s, such
-///   as API clients, analytics clients, random number generators, etc.
+///   * `State`: A type that holds the current state of the application.
```

Comment:
Just being consistent here. I added some code examples embedded in bullet points later on in this PR, which requires an indentation level of 8, so it looks best to make sure the text of bullets are indented by 4.